pkgs/by-name/az/az-pim-cli/version-build-info.patch

```diff
@@ -0,0 +1,25 @@
+diff --git a/cmd/version.go b/cmd/version.go
+index 816f044..7bbca7f 100644
+--- a/cmd/version.go
++++ b/cmd/version.go
+@@ -31,8 +31,8 @@ type BuildInfo struct {
+ }
+
+ func (b BuildInfo) String() string {
+-	return fmt.Sprintf("az-pim-cli version %s (built with %s from %s on %s)",
+-		b.Version, b.GoVersion, b.Commit, b.Date)
++    return fmt.Sprintf("az-pim-cli version %s (built with %s)",
++		b.Version, b.GoVersion)
```

Comment:
Go version currently prints unknown, should we either inject it via ldflags too or drop it here?